<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>432</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="all_actions">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>view all actions by the user sorted by end time</string>
   </property>
  </widget>
  <widget class="QPushButton" name="unique_applications">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>217</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>view a list of all applications run by the user</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
